commands/utils/quick-pick-image.ts

```diff
@@ -0,0 +1,37 @@
+import * as Docker from 'dockerode';
+import {docker} from './docker-endpoint';
+import vscode = require('vscode');
+
+
+
+export interface ImageItem extends vscode.QuickPickItem {
+    id: string,
+}
+
+function createItem(image: Docker.ImageDesc) : ImageItem {
+    return <ImageItem> {
+        label: image.RepoTags[0] || '<none>',
+        description: null,
+        id: image.Id
+    };
+}
+
+function computeItems(images: Docker.ImageDesc[]) : ImageItem[] {
+    let items : ImageItem[] = [];
+    for (let i = 0; i < images.length; i++) {
+        items.push(createItem(images[i]));
+    }
+    return items;
+}
+
+export function quickPickImage() : Thenable<ImageItem> {
```

Comment:
Would it make sense to allow removing all images in one command, similarly to how the stop container command works? That would be nice to not have to remember how to type `docker rmi $(docker images -qa)`!